boot/duneboot.ml

```diff
@@ -102,6 +102,8 @@ module List = struct
        | None -> find_map l ~f
        | Some _ as x -> x)
   ;;
+
+  let concat_map l ~f = List.map l ~f |> List.concat
```

Comment:
If you include `ListLabels` after this, we can pick the faster implementation when it's available (4.14).